.github/workflows/build_and_test_macos.yml

```diff
@@ -4,11 +4,14 @@ on:
   schedule:
     # daily at 3 AM
     - cron: "0 3 * * *"
+  pull_request_target:
```

Comment:
why not use pull_request here?

pull_request_target is generally avoided https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/

Reply:
Because I am an ignorant fool :) I will fix it.